src/chat/distantchat.cc

```diff
@@ -200,6 +201,7 @@ void DistantChatService::notifyTunnelStatus( const RsGxsTunnelId& tunnel_id, uin
         
     case RsGxsTunnelService::RS_GXS_TUNNEL_STATUS_REMOTELY_CLOSED:	RsServer::notify()->notifyChatStatus(ChatId(DistantChatPeerId(tunnel_id)),"Tunnel is down...") ;
         								RsServer::notify()->notifyPeerStatusChanged(tunnel_id.toStdString(),RS_STATUS_OFFLINE) ;
+        								RsServer::notify()->notifyDistantChatStatus(ChatId(DistantChatPeerId(tunnel_id))) ;
```

Comment:
I don't understand the goal of this line: it does exactly what the notifyChatStatus does 2 lines before: same tunnel  id, with message "tunnel is down". So there's no point in having both notify calls.